packages/ui/lib/RadioGroup/index.tsx

```diff
@@ -74,71 +30,105 @@ function useBaseRadioGroup({
   onChange,
   children,
 }: UseBaseRadioGroupProps) {
-  return {
-    // TODO: Should probably have the groupElem return a wrapperDiv so that it would
-    // be easy to for example change the Radios into a horizontal layout? Or maybe -
-    // this should be left to the consumer?
-    groupElem: (
-      <BaseRadioGroupContext.Provider value={{ activeValue: value, onChange }}>
-        {children}
-      </BaseRadioGroupContext.Provider>
-    ),
-  };
+  return (
+    <BaseRadioGroupContext.Provider value={{ activeValue: value, onChange }}>
+      {children}
+    </BaseRadioGroupContext.Provider>
+  );
 }
 
-export const Group = (props: UseBaseRadioGroupProps) => {
-  const { groupElem } = useBaseRadioGroup(props);
-  return groupElem;
+export const Group = ({
+  id,
+  className,
+  style,
+  ariaLabeledBy,
+  ...rest
+}: UseBaseRadioGroupProps & {
+  id?: string;
+  className?: string;
+  style?: CSSProperties;
+  ariaLabeledBy?: string;
+}) => {
+  const groupElem = useBaseRadioGroup(rest);
```

Comment:
Look at the other useBase hooks and copy the pattern from there. The return type could look like {
  buttonGroupProps,
  buttonGroupRef
}

And this must forwardRef